packages/preview/embiggen/0.0.1/demo.typ

```diff
@@ -0,0 +1,21 @@
+#import "src/lib.typ": *
```

Comment:
This import could use the package import to make it easier to start from this example.

```suggestion
#import "@preview/embiggen:0.0.1": *
```